u7-components/src/views/BookCard.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <router-link
+    :to="{ name: 'bookDetails', params: { isbn13: book.isbn13 } }"
+    class="book-card"
+  >
+    <div>
+     <img :src="`/images/books/${book.isbn13}.jpg`" :alt="book.title" />
+     <p>⭐{{ book.rating.toFixed(1) }}</p>
+     <p>{{ book.title }}</p>
+     <p>{{ book.price }}TH</p>
```

Comment:
คุณสามารถใช้ `{{ book.price.toLocaleString() }}` ได้หากต้องการคั่นด้วยเครื่องหมายจุลภาค
You can use `{{ book.price.toLocaleString() }}` if you want the comma separators.